src/template/index.js

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-param-reassign, no-underscore-dangle, import/no-extraneous-dependencies */
+const { Request } = require('node-fetch');
```

Comment:
wouldn't it be better to have our own Request/Response abstraction?
I don't like the extra dependency to node-fetch.

what about creating a `node-fetch-api` package that contains the _empty_ classes ?

Reply:
Yes, ideally something that just webpacks `Request` and `Response` from `node-fetch` and re-exports it. We can later switch transparently to `helix-fetch`